app/src/main/java/com/github/wnder/picture/FirebasePicturesDatabase.java

```diff
@@ -22,19 +24,43 @@
 import java.util.concurrent.CompletableFuture;
 
 import javax.inject.Inject;
+import javax.inject.Singleton;
 
+@Singleton
 public class FirebasePicturesDatabase implements PicturesDatabase {
 
     private final StorageReference storage;
     private final CollectionReference picturesCollection;
     private final CollectionReference usersCollection;
+    private final Context context;
```

Comment:
This doesn't need to be an attribute since it's only used in the constructor

Reply:
Done